docs/storage-management.asciidoc

```diff
@@ -123,30 +212,40 @@ POST /apm-*/_delete_by_query
   }
 }
 ------------------------------------------------------------
-// CONSOLE
 
 See {ref}/docs-delete-by-query.html[delete by query] for further information on this topic.
 
-[[delete-data-kibana]]
 [float]
+[[delete-data-kibana]]
 ==== Delete data via Kibana Index Management UI
-Select the indices you want to delete, and click the _Manage indices_ button to see the available actions.
-Then click _delete indices_.
-Follow the {kibana-ref}/managing-indices.html[Kibana Index Management] documentation
- to get started with the index management UI.
+
+Select the indices you want to delete, and click **Manage indices** to see the available actions.
+Then click **delete indices**.
+
+See {kibana-ref}/managing-indices.html[Kibana Index Management] for more information.
+
+[[manage-indices-kibana]]
+== Manage Indices via Kibana
+
+The Kibana UI for managing indices allows you to view indices, index settings,
+mappings, document counts, used storage per index, and much more.
+You can also perform management operations, like deleting indices directly via the Kibana UI.
+Finally, the UI supports applying bulk operations on several indices at once.
+Check out {kibana-ref}/managing-indices.html[Managing Indices] for more details.
```

Comment:
in case you didn't notice: same link 2 times in 9 lines.

Reply:
@bmorelli25 just a heads up, I am pretty sure most Elastic docs use an inline style for linking, rather than "see the {link}" or "check out {link}". Might want to verify with the rest of the writers if you're seeing a mix of approaches. 

*could be a great addition to our style guide